mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -81,7 +83,7 @@ public void start() {
         NetworkInfo activeNetworkInfo = connectivityManager.getActiveNetworkInfo();
         if (activeNetworkInfo != null) {
             if (settings.getBoolean(Constants.PREFERENCE_DEMOMODE, false)) {
-                mOpenHABUrl = mCtx.getString(R.string.openhab_demo_url);
+                mOpenHABUrl = "http://demo.openhab.org:8080/";
```

Comment:
https please

Reply:
Done, however, what do you think about my idea of getting rid of the OpenHABTracker, too? :D

Reply:
Right, I just saw http there.